### ISSUE TITLE
Can't see example in dashboard

### ISSUE THREAD

**Issue by cemsbr** [graphite-project/docker-graphite-statsd#40]
`echo counters | nc localhost 8126` shows the random data is being sent and I can see the example.wsp file, but the dashboard is empty and there's no metrics in the main page. What am I missing?

**Comment by cemsbr** [comment]
It worked after restarting the docker instance. If this is actually required, it should be in the README.

**Comment by deniszh** [comment]
It really depends on which volumes you are mounting. If your volume is empty this container populate it, but you need to restart container to pick up populated volume.

**Comment by cemsbr** [comment]
That was exactly what happened. Thanks for the clarification.

**Comment by samuelmanzer** [comment]
I hit this issue as well - if it is not too much trouble, would you mind clarifying what you said above:

"It really depends on which volumes you are mounting. If your volume is empty this container populate it, but you need to restart container to pick up populated volume."

Which volume does this refer to? I have these volumes shown by `docker inspect` (running on kubernetes):

```
            "Volumes": {
                "/etc/logrotate.d": {},
                "/etc/nginx": {},
                "/opt/graphite/conf": {},
                "/opt/graphite/storage": {},
                "/opt/graphite/webapp/graphite/functions/custom": {},
                "/opt/statsd": {},
                "/var/log": {}
            }
```

I think other folks may hit this as well because it happened to me when using the official [helm chart](https://github.com/helm/charts/tree/master/stable/graphite) out of the box, so figuring out if the chart should be changed or whether this step should go in the readme would be very helpful. Once I understand why this happens I'm happy to help with PRs etc

**Comment by deniszh** [comment]
PS: that's original init file for 0.9.15 - https://github.com/hopsoft/docker-graphite-statsd/blob/master/conf/etc/my_init.d/01_conf_init.sh
As you can see many things were added.

**Comment by deniszh** [comment]
Fixed in 1.1.4-1 release
I opened PR in helm charts for version upgrade - https://github.com/helm/charts/pull/7749
